### ISSUE TITLE
Quick improve to enable users returning to the app and increase adoption

### ISSUE THREAD

**Issue by palminha** [stayawayinesctec/stayaway-app#63]
Since the app runs in the background I tend to forget to enable/disable the app, or getting back to the app.

One quick win in order to get people back to the app could be an indication that would update automatically about how many "code" were exchanged or how many contacts you had.

This quick win would improve exponential people getting back to the app and use it more often!

**Comment by agamotto** [comment]
I actually received a notification from the Google module with those exact same stats.

**Comment by palminha** [comment]
@agamotto so it would be straightforward to show stats that would increase the user engagement with the App?

What kind of stats can we extract from the Google Module?

**Comment by davipt** [comment]
On the podcast https://umsobrezero.com/2020/09/21/episodio-33/ with the coordinator Prof. Rui Oliveira they talked about the subject that the ideal is for people to install the app and forget about it.

People thinking about the app could create anxiety about being afraid of the possible notification.

The app does not need to be manually started to trigger the background tasks it needs to do. In other words, there is no need to open the app for it to work. That is exactly why there was the effort from Apple and Google to provide the system APIs for this.

The other ideal scenario is for people to not complain that in certain situations it's possible that the battery drains a bit more, and the system (iOS) will tell them it's related to the Health app and the bluetooth token management. Killing the battery is what made people uninstall other similar apps.

In my opinion I fully agree with this, and therefore I also think there should be a split between this app, which should have a clear message of "install and forget about it", and a second independent app that would provide much better relevant and engaging information about the situation we're at right now… the EstamosOn Covid, which could be highly improved, not just the app itself, but primarily the relevant information that is hard to collect from the Min-Saude PDF and dashboard (arcgis) and that external teams like VOST PT and DSSG work so hard to process and deliver in a sensible format.

In other words, we need better information, but that's outside the scope of this app, IMHO. 
This app must have a clear message of "install and forget". People shall not stress out opening the app every day. It "just works" in the background.

**Comment by marado** [comment]
> On the podcast https://umsobrezero.com/2020/09/21/episodio-33/ with the coordinator Prof. Rui Oliveira they talked about the subject that the ideal is for people to install the app and forget about it.

There has been several "seemingly official" messages recommending against the "install and forget" approach, with a recomentation to check out the app daily. The Prime Minister said something to that effect ("I check my app every morning", or something of that kind). I do not recall where else (or how "official") I've seen about that, but a quick search took me to [a news item](https://www.dn.pt/dinheiro/app-stayaway-covid-perto-do-milhao-de-downloads-46-infetados-deixaram-alertas-12738010.html) that states that:

> por questões técnicas, convém ir visitando a app diariamente.

(ie, my translation:)

> for technical reasons, it is a good idea to visit the app daily.

**Comment by davipt** [comment]
> > On the podcast https://umsobrezero.com/2020/09/21/episodio-33/ with the coordinator Prof. Rui Oliveira they talked about the subject that the ideal is for people to install the app and forget about it.
> 
> There has been several "seemingly official" messages recommending against the "install and forget" approach, 

Yes I agree there's been a lot of misaligned communications that should not have happened and just created unnecessary noise. :( 

I wonder if this is related to the battery management issues on some android devices… I have to be honest and admit I only looked high level over the android code, and looked deep at the code and tested a lot on the iOS version (in particular with the iOS 14 betas), and it seems I incorrectly assumed the app would work as it does on iOS, where there is no need to open the app… unless there's a bug on a beta version lol

Still, it doesn't invalidate the idea that the *ideal* would be to install and forget. If there's a real need to take android users to open the app to ensure it still works, then I understand the original comment from @palminha

**Comment by fmaia** [comment]
Hi!
The intended design is indeed the "install and forget" approach. As @davipt mentioned, we have been, in some situations, recommending people to visit the app now and then. This is indeed targeted at certain Android devices that, due to their own tailored battery management systems, hinder the way the application works in the background. Visiting the app regularly mitigates this problem for these devices. We expect these to be a minority and thus have the "install and forget" scenario as the most common one.

**Comment by palminha** [comment]
The idea here is precisely to enhance that although the "install and
forget" can be the one that complies with a more process mindset, is not
the best from the user experience and user adoption point of view.

Every market expert and entrepreneur will tell you that apps are adopted by
users and not by processes...

As an open source project the big advantage is that it can follow the
collective intelligence of the community and not someone making all the
decisions inside an office

**Comment by davipt** [comment]
> Every market expert and entrepreneur will tell you that apps are adopted by users and not by processes

This is not a normal app. This is the equivalent to go to settings and enable "exposure notification" and hope to never hear from it again. People should know this app is not a funny game they want to open, but it's an app they do not want to use ever. "Adoption of installing" is not the same as "adoption of usage".

**Comment by palminha** [comment]
The app requires user adoption... Without adoption the app is useless.
